packages/compass-web/src/entrypoint.spec.tsx

```diff
@@ -97,16 +100,20 @@ describe('CompassWeb', function () {
       screen.getByText('Connecting to localhost:27017');
     });
 
-    expect(mockConnectFn.getCall(0).args[0].connectionOptions).to.have.property(
-      'connectionString',
-      'mongodb://localhost:27017/?appName=Compass+Web'
-    );
-
     await waitFor(() => {
       screen.getByText('Connected to localhost:27017');
     });
 
     expect(onTrackSpy).to.have.been.calledWith('New Connection');
+
+    const connectionId = onTrackSpy.firstCall.args[1][
```

Comment:
a bit weird to get it from there though has added benefit of making sure tracking + the URL are consistent.